docs/releases/4.1.txt

```diff
@@ -342,6 +342,11 @@ Templates
 * :tfilter:`json_script` template filter now allows wrapping in a ``<script>``
   tag without the HTML ``id`` attribute.
 
+* The :class:`cached template loader <django.template.loaders.cached.Loader>`
+  is now enabled in development, when :setting:`DEBUG` is ``True``, and
+  :setting:`OPTIONS['loaders'] <TEMPLATES-OPTIONS>` isn't specified. You may
+  specify ``OPTIONS['loaders']`` to override this, if necessary.
```

Comment:
I think you missed my [note](https://github.com/django/django/pull/15586#discussion_r849326930) about _"Backwards incompatible changes in 4.1"_ section.